app.bluebubbles.BlueBubbles.yml

```diff
@@ -0,0 +1,104 @@
+app-id: app.bluebubbles.BlueBubbles
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
+command: bluebubbles
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --device=dri
+  - --socket=pulseaudio
+  - --system-talk-name=org.freedesktop.NetworkManager
+  - --system-talk-name=org.freedesktop.NetworkManager.Connection.Active
+  - --system-talk-name=org.freedesktop.FileManager1
+  - --share=network
+  - --filesystem=xdg-data
+  - --filesystem=xdg-download
+modules:
+  - 'shared-modules/libappindicator/libappindicator-gtk3-12.10.json'
+  - 'shared-modules/dbus-glib/dbus-glib.json'
+  - name: objectbox
+    buildsystem: simple
+    build-commands:
+      - 'cp -r libobjectbox.so /app/lib/'
+    sources:
+      - type: archive
+        only-arches:
+          - x86_64
+        url: https://github.com/objectbox/objectbox-c/releases/download/v0.16.0/objectbox-linux-x64.tar.gz
+        sha256: b597c10607bbba71b238c935581ee7bde5ea07211d899bd4581685e495da01f8
+      - type: archive
+        only-arches:
+          - aarch64
+        url: https://github.com/objectbox/objectbox-c/releases/download/v0.16.0/objectbox-linux-aarch64.tar.gz
+        sha256: e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
+  - name: libidn
+    buildsystem: simple
+    build-commands:
+      - './configure --prefix=/app --disable-static && make && make install'
+    sources:
+      - type: archive
+        url: https://ftp.gnu.org/gnu/libidn/libidn-1.34.tar.gz
+        archive-type: tar-gzip
+        sha256: 3719e2975f2fb28605df3479c380af2cf4ab4e919e1506527e4c7670afff6e3c
+    cleanup:
+      - /bin/*
+      - /include/*
+      - /share/*
+  - name: vlc
+    buildsystem: simple
+    build-commands:
+      - 'BUILDCC="/usr/bin/gcc -std=gnu99" ./configure --prefix=/app --disable-a52 --disable-lua && make clean && make libcompat && make install'
+    sources:
+      - type: archive
+        url: http://download.videolan.org/pub/videolan/vlc/3.0.16/vlc-3.0.16.tar.xz
+        sha256: ffae35fc64f625c175571d2346bc5f6207be99762517f15423e74f18399410f6
+    cleanup:
+      - /bin/*
+      - /include/*
+      - /share/*
+      - /lib/vlc
+      - /lib/*.la
+  - name: bluebubbles
+    buildsystem: simple
+    build-commands:
+      - 'mkdir -p bluebubbles'
+      - 'tar -xf bluebubbles-linux-portable.tar.gz -C bluebubbles'
+      - 'cp -r bluebubbles /app/'
```

Comment:
This install a bunch of Windows DLL too. They should be skipped. See `data/flutter_assets/packages/record_windows`